.github/ISSUE_TEMPLATE/hotfix-template.md

```diff
@@ -0,0 +1,24 @@
+---
+name: Notify Hotfix Template
+about: Create hotfix ticket for urgent PRs
+title: 'HOTFIX: '
+labels: Notify, hotfix
```

Comment:
Should we prepopulate "patch"?  A hotfix doesn't typically add features.

Reply:
patch is for the pr _template_, maybe, this is just for the _ticket_ itself.